backend/src/main/java/com/auction/auctionapp/store/ProductSubCategoryRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.auction.auctionapp.store;
+
+import com.auction.auctionapp.model.Bid;
+import com.auction.auctionapp.model.Product;
+import com.auction.auctionapp.model.ProductSubCategory;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ProductSubCategoryRepository extends JpaRepository<ProductSubCategory, Integer> {
+    @Query(value = "SELECT * FROM ProductSubCategory WHERE productCategoryId = ?1", nativeQuery = true)
```

Comment:
Can we use named parameters inside a query - better readability, easier to modify query if later needed